includes/db/LoadBalancer.php

```diff
@@ -370,12 +370,13 @@ public function waitFor( $pos ) {
 	/**
 	 * Set the master wait position and wait for ALL slaves to catch up to it
 	 * @param $pos int
+	 * @param $wiki
 	 */
-	public function waitForAll( $pos ) {
+	public function waitForAll( $pos, $wiki ) {
```

Comment:
I don't know, the reason the other methods all have a default is because they are called in different ways. Sometimes with the argument passed in and sometimes not. This method is only called in one place and is always supplied an argument (since it's called by one of those methods with a default argument). I'm inclined to leave it as is until it's being used in more than one way in the code.